Kitodo/src/main/java/org/kitodo/services/data/ProcessService.java

```diff
@@ -928,10 +928,8 @@ public Fileformat readMetadataFile(Process process) throws ReadException, IOExce
         }
 
         Fileformat ff = determineFileFormat(type, process);
-
         try {
-            // TODO: this is not working anymore because uri is not full path
-            ff.read(metadataFileUri.toString());
+            ff.read(new File(metadataFileUri).toString());
```

Comment:
I think, thats the responsibility of the Java Garbage collector. We don't need to care about unused ressources.